src/lib/CRSF/devCRSF.cpp

```diff
@@ -22,10 +23,17 @@ static int timeout()
     return DURATION_IMMEDIATELY;
 }
 
+static int event()
+{
+    // An event should be generated every time the TX power changes
+    CRSF::LinkStatistics.uplink_TX_Power = POWERMGNT::powerToCrsfPower(POWERMGNT::currPower());
```

Comment:
I would say that the `powerToCrsfPower` also belongs over here in the CRSF library.